src/DynamoRevit/Properties/Resources.en-US.resx

```diff
@@ -151,4 +151,7 @@
   <data name="NewDocument" xml:space="preserve">
     <value>a new document.</value>
   </data>
+  <data name="BackgroundPreviewName" xml:space="preserve">
+    <value>Revit Background Preview</value>
+  </data>
```

Comment:
Maybe you could also add a comment like: `<comment>The name of the 3D background preview.</comment>`